server/odc-service/src/main/java/com/oceanbase/odc/service/db/browser/DBSchemaAccessors.java

```diff
@@ -98,11 +98,11 @@ public static DBSchemaAccessor create(@NonNull JdbcOperations syncJdbcExecutor,
             if (VersionUtils.isGreaterThanOrEqualsTo(obVersion, "4.0.0")) {
                 // OB 版本 >= 4.0.0
                 return new OBOracleSchemaAccessor(syncJdbcExecutor, new ALLDataDictTableNames());
-            } else if (VersionUtils.isGreaterThanOrEqualsTo(obVersion, "2.2.7")) {
-                // OB 版本为 [2.2.7, 4.0.0)
+            } else if (VersionUtils.isGreaterThanOrEqualsTo(obVersion, "2.2.70")) {
```

Comment:
It should be a bug if method calling `VersionUtils.isGreaterThan("2.2.30", "2.2.7")` returns true.
@yizhouxw may check this?